package.json

```diff
@@ -50,6 +50,7 @@
     "@tiptap/react": "^2.0.0-beta.108",
     "@tiptap/starter-kit": "^2.0.0-beta.183",
     "base64-arraybuffer": "^1.0.2",
+    "bvaughn-architecture-demo": "file:packages/bvaughn-architecture-demo",
```

Comment:
This is kind of gross, but I'm not sure if I want to go through making a new Workspace package _yet_, at least until Mark's done deciding what our new package manager will be.